be/src/main/java/yeonba/be/user/repository/favorite/FavoriteCommand.java

```diff
@@ -14,4 +14,8 @@ public Favorite save(Favorite favorite) {
 
     return favoriteRepository.save(favorite);
   }
+
+  public void delete(Favorite favorite) {
+    favoriteRepository.delete(favorite);
+  }
```

Comment:
저희가 정한 tapsize는 4입니다.
```suggestion
  public void delete(Favorite favorite) {
      favoriteRepository.delete(favorite);
  }
```